apps/web/src/components/routes/Dashboard/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Dashboard = () => (
```

Comment:
No esta mal utilizar const, solo para ser consistente con los componentes que agregamos antes, cambialo en los otros lados tambien.
```suggestion
function Dashboard(
```